content/docs/capabilities/routes-portal.mdx

```diff
@@ -34,16 +82,23 @@ This data is also available as JSON at `/.pomerium/api/v1/routes`.
 }
 ```
 
-### CLI
 
-The Pomerium CLI can list routes via the `routes list` subcommand:
+## FAQ
 
-```bash
-pomerium-cli routes list https://curious-cat-9999.pomerium.app
-```
+### Is the Routes Portal publicly available?
+Yes, the Routes Portal is publicly available, but only authenticated users will be able to see the routes they have access to.
+Unauthenticated users will be forced to authenticate.
 
-### Desktop Client
+### Can I disable the Routes Portal?
+No, the Routes Portal is a core feature of Pomerium and cannot be disabled. If you have concerns about the security of the portal, please contact security@pomerium.com.
```

Comment:
@gaurdro would you prefer this goes to support@?

Reply:
i'd prefer that go to security@pomerium.com  It may end up as a support request, but I think it would be prudent to triage it as a security inquiry first.